src/pcm.rs

```diff
@@ -619,6 +671,20 @@ impl<'a> HwParams<'a> {
         acheck!(snd_pcm_hw_params_get_buffer_time_max(self.0, &mut v, &mut d)).map(|_| v as u32)
     }
 
+    /// This function should only be called when the configuration space contains a single
```

Comment:
It seems a bit weird to have copied this additional comment but not the description of the function (i e "Check if hardware supports pause.").